### ISSUE TITLE
qpdf fails to merge large number of pages

### ISSUE THREAD

**Issue by jbarlow83** [jbarlow83/OCRmyPDF#181]
As described here qpdf failed to merge >1000 files due to hitting the ulimit of 256 on macos 
https://github.com/jbarlow83/OCRmyPDF/issues/117#issuecomment-319515364

**Comment by joewiz** [comment]
@jbarlow83 Thanks for opening a new issue for this. As you suggested, I've raised the limit to 2048 (`ulimit -n 2048`) - and have just started this batch bash command to let OCRmyPDF run overnight on ~10 such large PDFs (fingers crossed that at least one completes):

```bash
for PDF in $(find * -maxdepth 0 -type f ); do BASE=`basename $PDF .pdf`; ocrmypdf $PDF $BASE"_tesseract-ocr.pdf"; done;
```

I'll let you know how this goes!

**Comment by jbarlow83** [comment]
Fixed in v5.4.4
